src/views/HomeView.vue

```diff
@@ -1,39 +1,69 @@
 <template>
-  <div class="container" id="landingContainer">
-    <div class="span-image">
-      <img src="/public/images/field.jpg" class="landingImage">
-    </div>
-    <div id="textOverImage">
-      <div class="col titleArea">
-        <h1>Forward<br><span class="logoProgress">Progress</span></h1>
-      </div>
-    </div>
-    <div class="row">
-      <div class="col aboutRow">
-          <router-link to="about" class="aboutButton infoButton">Who We Are</router-link>
+  <div id="landingContainer">
+    <div class="featured-section">
+      <div class="row header3 mb-3">
+        <p>Leading the Way Forward in Football Analysis</p>
       </div>
+      <b-row align-h="center">
+        <b-col cols="4">
+          <b-carousel
+            id="landing-page-article-carousel"
+            :interval="4000"
+            fade
+            controls
+            indicators
+            background="#ababab"
+            img-width="100%"
+            img-height="400px"
+            v-model="currentSlide"
+          >
+            <b-carousel-slide v-for="post in carouselSlides" v-bind:key="post.id"
```

Comment:
carouselSlides is a computed property that is always the first 3 posts in `latestPosts`